include/swift/Runtime/MutexWASI.h

```diff
@@ -0,0 +1,67 @@
+//===--- MutexWASI.h - -----------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Stub implementation of Mutex, ConditionVariable, Read/Write lock, and Scoped 
```

Comment:
s/stub/no-op/

I wouldn't say "no concrete implementation is provided" because it is a concrete implementation, it just does not perform locking.

"No-op implementation of locks for the WebAssembly System Interface. The implementation does not need to perform locking, because as of <insert WebAssembly spec version, or today's date> WebAssembly does not support threads."